options.js

```diff
@@ -1,32 +1,35 @@
 var tcDefaults = {
-  speed: 1.0,           // default 1x
-  speedStep: 0.1,       // default 0.1x
-  rewindTime: 10,       // default 10s
-  advanceTime: 10,      // default 10s
-  fastSpeed: 1.8,       // default 1.8x
-  resetKeyCode:  82,    // default: R
-  slowerKeyCode: 83,    // default: S
-  fasterKeyCode: 68,    // default: D
-  fastKeyCode: 71,      // default: G
-  rewindKeyCode: 90,    // default: Z
-  advanceKeyCode: 88,   // default: X
+  speed: 1.0,           // default:
   displayKeyCode: 86,   // default: V
   rememberSpeed: false, // default: false
   startHidden: false,   // default: false
+  keyBindings: [
+    ["slower", 83, 0.1, 0], // default: S 0.1
+    ["faster", 68, 0.1, 0], // default: D 0.1
+    ["rewind", 90, 10, 0], // default: Z 0.1
+    ["advance", 88, 10, 0], // default: X 0.1
+    ["reset", 82, 1, 0], // default: R 1
+    ["fast", 71, 1.8, 0] // default: G 1.8
+  ],
+  version: "0.5.3",     // default: 0.5.3
```

Comment:
why do we need to mirror the version number here?

Reply:
on line 43 `if (!storage.version) // if version is lower than 0.5.2 or first initialization` we need to use something to test if it's first upgrade. maybe we can try if we have keyBindings in storage

Reply:
Hmm, I don't see that line?

If we do want to persist the version, we can use `chrome.runtime.getManifest().version` to query the version in the manifest, instead of hardcoding it.